src/server/twemcache-rs/src/common.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2021 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+use crate::Config;
+
+use boring::ssl::{SslAcceptor, SslContext, SslFiletype, SslMethod};
```

Comment:
With the understanding that we will (soon) consolidate common event/IO logic between different servers, I'm skipping this file.